Vic2ToHoI4/Data_Files/Configurables/names.txt

```diff
@@ -0,0 +1,7071 @@
+#The ones listed below are taken from HoI4 Vanilla \common\names\00_names.txt
```

Comment:
This file is upload to a capital C Configurables folder instead of the existing lower case one. This has confused github and will cause more problems in the future. Please rename the folder.